api/api_ethereum.go

```diff
@@ -1225,8 +1225,6 @@ func (args *EthTransactionArgs) ToMessage(globalGasCap uint64, baseFee *big.Int,
 			gasPrice = args.GasPrice.ToInt()
 		} else if args.MaxFeePerGas != nil {
 			gasPrice = args.MaxFeePerGas.ToInt()
-		} else {
```

Comment:
Can you fix L1223 comment? I think this if condition includes EthTxTypeCompatible hard fork (support EIP-1559) also.

Reply:
Thanks!